bin.src/write_amp_image_file.py

```diff
@@ -11,11 +11,17 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("eimage_file", help="eimage file to process")
+parser.add_argument('--log_level', type=str,
```

Comment:
Just to check to make sure I know what is going on: This work on adding loggers is just to add this functionality (which exists in imsim.py) to the electronics code correct? I mean there is nothing specific about the header keywords and it could in principle be in it's own PR.

Reply:
right, nothing keyword specific here.  didn't seem worth the trouble to make a separate branch and pr.